js/jquery.markitup.js

```diff
@@ -405,12 +404,12 @@
 					textarea.setSelectionRange(start, start + len);
 				}
 				textarea.scrollTop = scrollPosition;
-				textarea.focus();
+				setTimeout(function(){ textarea.focus(); }, 1);
```

Comment:
Is 1ms enough? Will it work on slower machines? I presume whole is a bit *hacky*.